docs/references/architecture/tendermint-core/adr-053-state-sync-prototype.md

```diff
@@ -211,15 +211,15 @@ Snapshots must also be garbage collected after some configurable time, e.g. by k
 
 * Do we need incremental chunk verification for first version?
 
-    > No, we'll start simple. Can add chunk verification via a new snapshot format without any breaking changes in Tendermint. For adversarial conditions, maybe consider support for whitelisting peers to download chunks from.
+    > No, we'll start simple. Can add chunk verification via a new snapshot format without any breaking changes in Tendermint. For adversarial conditions, maybe consider support for allowlisting peers to download chunks from.
```

Comment:
Tendermint Core docs are here for historical purposes. I think they should not be changed.